.vscode/settings.json

```diff
@@ -27,5 +27,18 @@
     "python.formatting.provider": "yapf",
     "[json]": {
 	    "editor.formatOnSave": false
-    }
+    },
```

Comment:
please revert.

Reply:
Not sure how I changed this. I don't remember opening the file. I think it has been reverted now.